packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -945,3 +976,429 @@ class _RenderTheatre extends RenderBox with ContainerRenderObjectMixin<RenderBox
     ];
   }
 }
```

Comment:
we should update `Positioned`'s API docs to mention that `Overlay` also uses `Positioned`.

Reply:
https://github.com/flutter/flutter/issues/107903. Tried to update in this pull request but I couldn't find the right way to phrase it.